utils/auditd/README.md

```diff
@@ -0,0 +1,11 @@
+auditd
+------
+
+Q: What is this?
+A: A simple audit daemon that throws away all messages.
+
+Q: Is it used anywhere?
+A: Yes. Lorax templates replace the actual well-known auditd with our stub.
+
+Q: Why?
+A: We don't know anymore, it's a cargo cult.
```

Comment:
It would be nice to fill this hole in the knowledge.

(@bcl, would you happen to remember?)

Reply:
I had to ask pjones -- I had totally forgotten that was there. This was written in 2006 when the available space was much smaller. But it is probably still valid. auditd takes up a bunch of space, it doesn't really provide anything useful in the installation environment, and would likely clutter up the logs with unnecessary messages.